Error building backend on node v16.15.0

<!--
   SUPPORT REQUESTS:
   This is for reporting bugs in Mempool, not for support requests. 
   If you have a support request, please join our Keybase or Matrix:
   https://keybase.io/team/mempool
   https://matrix.to/#/#mempool:bitcoin.kyoto
-->

### Description
building the backend on a fresh git clone fails.
<!-- brief description of the bug -->

#### Version
https://github.com/mempool/mempool/commit/da3c7588b86cd1287364be9e7123c304fece8549
node: v16.15.0 (Latest LTS)
npm: 8.5.5
<!-- commit id or version number -->

### Steps to reproduce
git clone mempool, cd backend, npm install --prod, npm run build, error
<!-- if you can reliably reproduce the bug, list the steps here -->

#### Device or machine
debian 11, amd64
<!-- device/machine used, operating system -->

#### Additional info
Original Error: https://pastebin.com/tvaRip7C
<!-- Additional information useful for debugging (e.g. logs) -->

I also tried the suggestion in the error and did `npm i --save-dev @types/node`:
After that the npm run build:
```
npm run build

> mempool-backend@2.4.0-dev build
> npm run tsc


> mempool-backend@2.4.0-dev tsc
> ./node_modules/typescript/bin/tsc

src/api/disk-cache.ts:20:18 - error TS2339: Property 'isMaster' does not exist on type 'typeof import("cluster")'.

20     if (!cluster.isMaster) {
                    ~~~~~~~~

src/index.ts:47:17 - error TS2339: Property 'isMaster' does not exist on type 'typeof import("cluster")'.

47     if (cluster.isMaster) {
                   ~~~~~~~~

src/index.ts:53:32 - error TS2339: Property 'fork' does not exist on type 'typeof import("cluster")'.

53         const worker = cluster.fork(env);
                                  ~~~~

src/index.ts:57:15 - error TS2339: Property 'on' does not exist on type 'typeof import("cluster")'.

57       cluster.on('exit', (worker, code, signal) => {
                 ~~

src/index.ts:62:37 - error TS2339: Property 'fork' does not exist on type 'typeof import("cluster")'.

62           const newWorker = cluster.fork(env);
                                       ~~~~

src/index.ts:109:73 - error TS2339: Property 'isMaster' does not exist on type 'typeof import("cluster")'.

109     if (config.STATISTICS.ENABLED && config.DATABASE.ENABLED && cluster.isMaster) {
                                                                            ~~~~~~~~


Found 6 errors in 2 files.

Errors  Files
     1  src/api/disk-cache.ts:20
     5  src/index.ts:47
```
Frontend builds fine btw

--- a/backend/README.md
+++ b/backend/README.md
@@ -77,13 +77,13 @@ Query OK, 0 rows affected (0.00 sec)
 
 #### Build
 
-_Make sure to use Node.js 16.15 and npm 7._
+_Make sure to use Node.js 16.10 and npm 7._
 
 Install dependencies with `npm` and build the backend:
 
 ```
 cd backend
-npm install # add --prod for production
+npm install
 npm run build
 ```
 

--- a/frontend/README.md
+++ b/frontend/README.md
@@ -34,7 +34,7 @@ $ npm run config:defaults:bisq
 
 ### 3. Run the Frontend
 
-_Make sure to use Node.js 16.15 and npm 7._
+_Make sure to use Node.js 16.10 and npm 7._
 
 Install project dependencies and run the frontend server:
 
@@ -71,13 +71,13 @@ Set up the [Mempool backend](../backend/) first, if you haven't already.
 
 ### 1. Build the Frontend
 
-_Node.js 16 and npm 7 are recommended._
+_Make sure to use Node.js 16.10 and npm 7._
 
 Build the frontend:
 
 ```
 cd frontend
-npm install # add --prod for production
+npm install
 npm run build
 ```
 

